integration-tests/elytron-undertow/src/test/java/io/quarkus/it/undertow/elytron/BaseAuthRestTest.java

```diff
@@ -0,0 +1,43 @@
+package io.quarkus.it.undertow.elytron;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.is;
+
+import org.junit.jupiter.api.RepeatedTest;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.http.ContentType;
+
+@QuarkusTest
+class BaseAuthRestTest {
+
+    @Test
+    @RepeatedTest(100)
+    void testPost() {
+        // This is a regression test in that we had a problem where the Vert.x request was not paused
+        // before the authentication filters ran and the post message was thrown away by Vert.x because
+        // RESTEasy hadn't registered its request handlers yet.
```

Comment:
This comment looks off?

Reply:
We don't have anywhere that is specifically dedicated to testing RESTEasy + Undertow, so I just enhanced the security tests to also test RESTEasy on Undertow, but used this mechanism to map RESTeasy.